### ISSUE TITLE
Possible bug? Acrylic can not be enabled for Ubuntu profiles

### ISSUE THREAD

**Issue by andrerav** [microsoft/terminal#9938]
Hi,

I don't know if this is a bug or a known limitation (I tried googling but couldn't find anything). Anyway, I'll try to keep this short and simple. I am trying to enable acrylic for my Ubuntu profile:

![image](https://user-images.githubusercontent.com/3635018/115958087-33a88100-a506-11eb-96f9-9753bcf9ee68.png)

But when I save my changes the acryclic switch is reset to off:

![image](https://user-images.githubusercontent.com/3635018/115958098-428f3380-a506-11eb-9e91-3f231bb8c2c8.png)

Acryclic does seem to work just fine for Powershell and CMD. This happens on both of my computers. I am using the Windows Terminal Preview from the Microsoft Store. If anyone knows what is causing this issue then please let me know :)

**Comment by skyline75489** [comment]
What is the version you're using? Please fill the template to provide the necessary information.

**Comment by zadjii-msft** [comment]
Yea, the specific version number would help here. But we have been seeing a good number of these "I change a setting and it didn't apply" issues - see #9855, #9864. There's something weird about layering and the SUI, that's for sure.

**Comment by zadjii-msft** [comment]
We've managed to track down a minimal repro of this. We're now tracking this in #9962. Thanks for the help!

/dup #9962

**Comment by andrerav** [comment]
Sorry for not following up! The exact version I am using _right now_ is 1.8.1032.0, however the version I was using when reporting the bug above was Windows Terminal Preview installed from Windows Store the same day as I reported the bug. However it seems the issue is gone now and I can apply the acrylic effect to Ubuntu. I don't know if it's because the app has been updated, or if it's because there are some specific factors that affects this. Let me know if there's anything else I can help with :)
